config.go

```diff
@@ -44,10 +44,20 @@ var (
 	effectiveConfigGOPATHs []string
 )
 
+const srclibGopath = ".srclib-gopath"
```

Comment:
Is the "current working directory" of srclib-go guaranteed to be user's home directory? This seems to imply that, but I'm not sure if it's the case.